plaso/parsers/winreg_plugins/run.py

```diff
@@ -84,7 +84,9 @@ def ExtractEvents(self, parser_mediator, registry_key, **kwargs):
       entries.append(value_string)
 
     event_data = RunKeyEventData()
-    event_data.entries = ' '.join(sorted(entries)) or None
+    # If entries is empty make sure it is set to an empty sting otherwise
```

Comment:
string, not sting

Reply:
Not sure what you mean can you be more specific.

Reply:
per conversation you meant there was a typo in "sting"